README.md

```diff
@@ -11,6 +11,15 @@
 
 This project is the official [Terraform Provider](https://registry.terraform.io/providers/stackitcloud/stackit/latest/docs) for [STACKIT](https://www.stackit.de/en/), which allows you to manage STACKIT resources through Terraform.
 
+> [!WARNING]
+> On August 26 2024, The STACKIT Argus service was renamed to STACKIT Observability.
+>
+> This means that there are new resources for `STACKIT Observability`, which offers the same functionality as the deprecated `STACKIT Argus` resources.
+>
+> Please make sure to **update your STACKIT Terraform Provider to the latest version after August 26 2024** to ensure that you start using STACKIT Observability resources.
+> 
+> You can migrate your existing STACKIT Argus resources by using the `terraform import`. https://developer.hashicorp.com/terraform/language/import
+

```

Comment:
I think it's enough to add this message to the changelog of the release instead of the README

Reply:
Also we should probably also give a hint on how to remove the existing Argus resource without destroying it: remove from TF config and remove from state (`terraform state rm stackit_argus_instance.RESOURCE_NAME`)

Reply:
removed the info from readme and left a comment in the work item to add it in the release changelog. I will also add the second part in theere.